README.md

```diff
@@ -64,6 +64,14 @@ pipenv install --dev
 pytest --cov=ros tests
 ```
 
+## Running Inventory API with xjoin pipeline
+To run full inventory api with xjoin , run the following command:
+```bash
+docker-compose up insights-inventory-web xjoin
```

Comment:
@patilsuraj767 Can we combine the docker-compose command?

`docker-compose up insights-inventory-mq db-ros insights-engine insights-inventory-web xjoin`

The containers should set up in order and seems to work for me.

Reply:
`xjoin` is only required when working with local frontend. Everytime starting xjoin while backend development will consume more resources. That is the reason I kept it separate so that we only start xjoin while working with local frontend.

Reply:
Makes sense, thanks!